--- Marlin/Configuration.h	2017-05-08 12:25:10.596782049 +0200
+++ ../Configuration.h	2017-05-08 12:39:37.079926193 +0200
@@ -476,14 +476,14 @@
  * Override with M92
  *                                      X, Y, Z, E0 [, E1[, E2[, E3[, E4]]]]
  */
-#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 4000, 750 }
+#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80.19, 80.19, 400, 96 }
 
 /**
  * Default Max Feed Rate (mm/s)
  * Override with M203
  *                                      X, Y, Z, E0 [, E1[, E2[, E3[, E4]]]]
  */
-#define DEFAULT_MAX_FEEDRATE          { 300, 150, 2.5, 25 }
+#define DEFAULT_MAX_FEEDRATE          { 250, 150, 15 , 80 }
 
 /**
  * Default Max Acceleration (change/s) change = mm/s
@@ -491,7 +491,7 @@
  * Override with M201
  *                                      X, Y, Z, E0 [, E1[, E2[, E3[, E4]]]]
  */
-#define DEFAULT_MAX_ACCELERATION      { 2000, 1000, 100, 10000 }
+#define DEFAULT_MAX_ACCELERATION      { 2000, 900, 100, 10000 }
 
 /**
  * Default Acceleration (change/s) change = mm/s
@@ -501,7 +501,7 @@
  *   M204 R    Retract Acceleration
  *   M204 T    Travel Acceleration
  */
-#define DEFAULT_ACCELERATION          2500    // X, Y, Z and E acceleration for printing moves
+#define DEFAULT_ACCELERATION          2000    // X, Y, Z and E acceleration for printing moves
 #define DEFAULT_RETRACT_ACCELERATION  3000    // E acceleration for retracts
 #define DEFAULT_TRAVEL_ACCELERATION   3000    // X, Y, Z acceleration for travel (non printing) moves
 
