--- Marlin/Configuration_adv.h	2017-05-08 12:25:10.596782049 +0200
+++ ../Configuration_adv.h	2017-05-08 12:45:29.787926193 +0200
@@ -212,8 +212,8 @@
  * Multiple extruders can be assigned to the same pin in which case
  * the fan will turn on when any selected extruder is above the threshold.
  */
-#define E0_AUTO_FAN_PIN -1
-#define E1_AUTO_FAN_PIN -1
+#define E0_AUTO_FAN_PIN  6
+#define E1_AUTO_FAN_PIN  8
 #define E2_AUTO_FAN_PIN -1
 #define E3_AUTO_FAN_PIN -1
 #define E4_AUTO_FAN_PIN -1
